src/app/components/App.jsx

```diff
@@ -331,12 +330,11 @@ export default connect(
     },
     dispatch => ({
         loginUser: () =>
-            dispatch(user.actions.usernamePasswordLogin()),
+            dispatch(userActions.usernamePasswordLogin({})),
```

Comment:
The change from user.actions* to userActions was to preserve `redux/User` imports?

Reply:
yes -- redux-modules enforced the actions child object